olp-cpp-sdk-dataservice-read/src/VersionedLayerClientImpl.cpp

```diff
@@ -672,8 +698,8 @@ client::CancellationToken VersionedLayerClientImpl::GetAggregatedData(
   auto settings = settings_;
   auto lookup_client = lookup_client_;
 
-  auto data_task =
-      [=](client::CancellationContext context) -> AggregatedDataResponse {
+  auto data_task = [=](client::CancellationContext context) mutable
```

Comment:
mutable for what?

Reply:
As you saw I've fixed quite a few wrong or absent usages of 'std::move'. This is one of them: you can't move captured by value objects inside a non-mutable lambda